docs/pages/preview/eas-update/deployment-patterns.md

```diff
@@ -0,0 +1,152 @@
+
+Publishing updates: (c) create update branches that are environment-based, like "staging" and "production".
+
+Diagram of flow:
+
+![Staging deployment diagram](/static/images/eas-update/deployment-staging.png)
+
+Explanation of flow:
+
+- Develop a project locally and test changes in Expo Go.
+- Create builds with channels named "production", which will eventually get reviewed and become available on app stores. Create another set of builds with channels named "staging", which will be used for testing on TestFlight and the Play Store Internal Track.
+- Set up `expo-github-action` to publish updates when merging commits to branches.
+- Merge changes into a branch named "staging". The GitHub Action will publish an update and make it available on our test builds.
+- When ready, merge changes into the "production" branch to publish an update to our production builds.
+
+Advantages of this flow:
+
+- This flow allows you to control the pace of deploying to production independent of the pace of development. This adds an extra chance to test your app and avoids your user having to download a new update every time a PR is landed.
+- It's easy to communicate to your team, since deploying updates occurs when merging into GitHub branches named "staging" and "production".
+
+Disadvantages of this flow:
+
+- Checking out previous versions of your app is slightly more complex, since we'd need to check out an old commit instead of an old branch.
+- When merging to "production", the update would be re-built and re-published instead of moved from the builds with channel "staging" to the builds with channel "production".
+
+## Platform-specific flow
+
+This flow is for projects that need to build and update their Android and iOS apps separately all the time. It will result in separate commands for delivering updates to the Android and iOS apps. Here are the parts of the deployment process above that make up this flow:
+
+Creating builds: (a) Create builds for production use only, or (b) create builds for production use and separate builds for testing use.
+
+Testing changes: (a) Test changes on TestFlight and the Play Store Internal Track and/or (b) Test changes with internal distribution builds.
+
+Publishing updates: (c) create update branches that are environment- and platform-based, like `ios-staging`, `ios-production`, `android-staging`, and `android-production`.
+
+Diagram of flow:
+
+![Platform specific deployment diagram](/static/images/eas-update/deployment-platform-specific.png)
+
+Explanation of flow:
+
+- Develop a project locally and test changes in Expo Go.
+- Create builds with channels named like `ios-staging`, `ios-production`, `android-staging`, and `android-production`. Then put the `ios-staging` build on TestFlight and submit the `ios-production` build to the public App Store. Likewise, put the `android-staging` build on the Play Store Internal Track, and submit the `android-production` build to the public Play Store.
```

Comment:
Everywhere else (?) channel names are double-quoted instead of code-formatted:
```suggestion
- Create builds with channels named like "ios-staging", "ios-production", "android-staging", and "android-production". Then put the "ios-staging" build on TestFlight and submit the "ios-production" build to the public App Store. Likewise, put the "android-staging" build on the Play Store Internal Track, and submit the "android-production" build to the public Play Store.
```